src/global/types.h

```diff
@@ -1741,6 +1756,8 @@ typedef struct CAMERA_INFO {
     /* 0058 */ ITEM_INFO *last_item;
     /* 005C */ OBJECT_VECTOR *fixed;
     /* 0060 end */
+    int16_t additional_angle;       //used for the manual camera controll
+    int16_t additional_elevation;
```

Comment:
Consider renaming these to `angle_override` and `elevation_override`. Please put the comment on a separate line.